FinniversKit.xcodeproj/project.pbxproj

```diff
@@ -1428,7 +1435,8 @@
 				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
 				PRODUCT_BUNDLE_IDENTIFIER = no.finn.FinniversKit;
-				PRODUCT_NAME = "$(TARGET_NAME:c99extidentifier)";
```

Comment:
What is this?

Reply:
I think this is required for Carthage to build a .framework

Reply:
Ok, I will delete this PR/ branch. We can have a discussion about it later :)